### ISSUE TITLE
import type is not recognized by lit-analyzer

### ISSUE THREAD

**Issue by Christian24** [runem/lit-analyzer#137]
Hello,

we just stumbled upon the issue that lit-analyzer treats [type-only imports](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export) as regular imports. 
Consider the following example: 
````typescript
import type {ComponentA} from 'foo/componentA' // defines <component-a>
const component: ComponentA = document.queryselector('component-a')
const template = html`<component-a></component-a>`
````
This should trigger the missing-import-rule. However, at the moment it does not. 

Thanks,
Christian
